athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/data/BlockUtilsTest.java

```diff
@@ -156,4 +158,17 @@ public void fieldToString()
         assertTrue(actual.contains(expectedCol1));
         assertTrue(actual.contains(expectedCol2));
     }
+
+    @Test
+    public void issue147()
```

Comment:
can we give this a name that stands on its own independent of the GitHub issue numer?

Reply:
I renamed it. If you have any idea for good name, please let me know.